### DISCUSSION TITLE
Hover from the React testing library in the Tooltip component is not working

### DISCUSSION THREAD

**Discussion by altzprogger** [chakra-ui/zag#2133] [Q&A] [answered]
Hello everyone. I'm trying to write a test using the React Testing Library to view a hint on hover. When using hover from UserEvent zag.js does not change the state of the Tooltip in any way, that is, the tooltip does not open. I also tried to use fireEvent using the MouseEvent('mouseover') event and used fireEvent.focus(), but the situation persists. What to do?

**Comment by segunadebayo** [comment, marked as answer]
Please use this code as reference
https://github.com/chakra-ui/ark/blob/main/packages/react/src/components/tooltip/tests/tooltip.test.tsx
